tests/test_optimizer.py

```diff
@@ -285,6 +285,21 @@ def test_qualify_columns__with_invisible(self):
         schema = MappingSchema(self.schema, {"x": {"a"}, "y": {"b"}, "z": {"b"}})
         self.check_file("qualify_columns__with_invisible", qualify_columns, schema=schema)
 
+    def test_pushdown_cte_alias_columns(self):
+        # Both of these should have SUM(a) aliased to "c"
+        queries = [
```

Comment:
Let's just create another fixture file. We can add more tests for unions, subqueries, etc.

Reply:
Just added it.